### DISCUSSION TITLE
Is this project dead?

### DISCUSSION THREAD

**Discussion by juparker37** [hvac/hvac#1188] [General]
With 134 issues open, no responses to opened issues in May 2024, looking for maintainers, is the project founder MIA?

**Comment by briantist** [comment]
## No, it's not dead,
but it could go that way, this is the nature of open source.

The project founder hasn't been involved many years AFAIK, right now I'm the only active maintainer and as you've seen, I've become less active. This is due to a number of factors, but it's mostly the usual reasons: this is a side project for me (one of several), I have a day job, and a personal life, and time is a precious commodity.

I also opened the [discussion looking for contributors of all types](https://github.com/hvac/hvac/discussions/994) because I care about projects being healthy and sustainable from a community standpoint, and because I recognized even then that having more help would be needed.

We get some contributions here and there but no one has shown sustained interest in taking a larger role yet.

That's just how these things go. 

Hopefully I will find myself with more time to put toward this again, if not hopefully there will be people willing to take a more active role by then and I can put what little time I have into supporting them, time will tell!
